pkg/apis/apis/v1alpha1/types.go

```diff
@@ -346,6 +346,14 @@ type APIExportSpec struct {
 	//
 	// +optional
 	Identity *Identity `json:"identity"`
+
+	// TODO: before beta we should re-evaluate this field name
+
+	// maximalPermissionPolicy will allow for a service provider to set a upper bound on what is allowed
+	// for a consumer of this API. If the policy is not set, no upper bound is applied,
+	// i.e the consuming users can do whatever the user workspace allows the user to do.
+	// +optional
+	MaximalPermissionPolicy *APIExportPolicy `json:"maximalPermissionPolicy,omitempty"`
```

Comment:
@ncdc any ideas for a better field name? Feels like we didn't get it quite right yet (the cosmetical part).

Reply:
I'm ok on the current name. We can bike shed for a future alpha & definitely before beta